contents.json

```diff
@@ -17151,6 +17151,23 @@
       ],
       "date_added": "Nov 18 2018",
       "suggested_by": "@dkhamsing"
-    }
+    },
+    {
+      "title": "WeAre",
+      "category-ids": ["photo"],
+      "description": "photo album of AR",
+      "source": "https://github.com/SherlockQi/HeavenMemoirs",
+      "homepage": "https://github.com/SherlockQi",
+      "itunes": "https://itunes.apple.com/app/id1234567890",		
```

Comment:
@SherlockQi replace the iTunes link by this `https://itunes.apple.com/app/weare/id1304227680` link, please.